src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -199,13 +240,13 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
     const questionIds: Array<string> = flattenDeep(nestedIds);
     return questionIds.includes(idToTest);
   };
-
   const handleCreateQuestion = () => {
-    createQuestion();
+    const updatedAnswers = handleSaveMoreAnswers();
+    createQuestion(updatedAnswers);
     closeModal();
   };
 
-  const createQuestion = () => {
+  const createQuestion = (selectedAnswers) => {
```

Comment:
Its better to rename this parameter since we have a state with the same name

Reply:
And the update the newQuestion object to use the passed in argument to set the answers